lib/membrane_element_tee/parallel.ex

```diff
@@ -1,17 +1,14 @@
 defmodule Membrane.Element.Tee.Parallel do
-  use Membrane.Filter
-
   @moduledoc """
-  Element for forwarding packets to two or more outputs
-
-  There is one input pad `:input` and output pad:
+  Element for forwarding packets to multiple outputs
 
-  * `:output` - is a dynamic pad which is available on request and works in pull mode
-
-  Basically we can forward packets to more than one destination by linking dynamic pad to one or more inputs. Packets are forwarded only when all output pads send demands.
-  To avoid overriding your links to output pads, create them with specific id ({:tee, :output, id})
+  To use, link this element to one preceding element via `input` and multiple
+  succesive elements via `output`. Each buffer is forwarded only when demand for
+  it comes in via each output.
```

Comment:
You can mention the result - that processing speed will be limited by the slowest consumer